cli/polka-storage-provider/client/src/commands/proofs.rs

```diff
@@ -72,6 +77,15 @@ pub enum ProofsCommand {
         /// Directory where the proof files and the sector will be put. Defaults to the current directory.
         #[arg(short, long)]
         output_path: Option<PathBuf>,
+        /// Sector number
+        #[arg(long)]
+        sector_id: u32,
+        /// The height at which we draw the randomness for deriving a sealed cid.
```

Comment:
We should propose the defaults that were hardcoded here, If I were a new user, I wouldn't know where to put it and which number needs to be after which one.

Reply:
I thought that we will eventually remove those commands? The defaults were also specific for a single use case